Userland/Libraries/LibThreading/Thread.cpp

```diff
@@ -46,3 +46,14 @@ void Threading::Thread::start()
     }
     dbgln("Started thread \"{}\", tid = {}", m_thread_name, m_tid);
 }
+
+void Threading::Thread::detach()
+{
+    if (m_detached)
+        return;
```

Comment:
Should we instead `VERIFY()` this instead? If someone calls `detach()` twice that's almost certainly a bug, isn't it?

Reply:
Sounds sensible to me, I'll get that updated